### PR TITLE
Add info to login and move logout

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by tsauerwein** [comment]
Looks fine to me

**Comment by asaunier** [comment]
Except my comment about the roles, OK for me as well.

**Comment by gberaudo** [comment]
I updated the PR with your comments. Is it OK for you @tsauerwein and @asaunier?

**Comment by tsauerwein** [comment]
:+1:

File: c2corg_api/views/user.py [hunk 1]
```diff
@@ -107,11 +108,31 @@ def post(self):
...
+        token = try_login(user, password, request) if user else None
         if token:
+            expire_time = token.expire - datetime.datetime(1970, 1, 1)
+            roles = ['moderator'] if user.moderator else []
             return {
-                'token': token
+                'token': token.value,
+                'username': user.username,
+                'expire': int(expire_time.total_seconds()),
+                'moderator': roles
```
Comment (asaunier) [inline c2corg_api/views/user.py]: I think you should rename 'moderator' to 'roles'.

**Comment by asaunier** [comment]
One small comment, else feel free to merge.

**Comment by gberaudo** [comment]
Thanks for the review.

**Pull request merged**